deploy/docker-compose/.env-sample

```diff
@@ -1,7 +1,9 @@
-# your instance public hostname
-HOOP_PUBLIC_HOSTNAME=
-# enabled|disabled
-HOOP_TLS_MODE=
+# In case you are running the application locally without a custom Identity Provider,
```

Comment:
Did you test it with a custom identity provider to see if it works?

Reply:
works like a charm!